examples/nlp/language_modeling/conf/megatron_falcon_inference.yaml

```diff
@@ -0,0 +1,39 @@
+inference:
```

Comment:
We have many `megatron_*_inference.yaml ` configs with minor differences. We could setup some common denominator for them so that it becomes more transparent and manageable (especially that more and more configs are expected to be added in future). 

That's a general mark cc @ericharper